lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
```

Comment:
thank you, too.

if we create `build` and `update` options, how should the below work?
```ruby
carthage(
  build: true,
  update: true
)
```
`command` option clarifies what the command to be executed, while `build` and `update` options need implicit priority.

For example, 
**gym**
```ruby
def self.define_profile_type(values)
  profile_type = "app-store"
  profile_type = "ad-hoc" if values[:type] == 'adhoc'
  profile_type = "development" if values[:type] == 'development'
  profile_type = "enterprise" if values[:type] == 'enterprise'

  Helper.log.info "Setting Provisioning Profile type to '#{profile_type}'"

  Actions.lane_context[SharedValues::SIGH_PROFILE_TYPE] = profile_type
end
```
**sigh**
```ruby
def self.set_profile_type(values, enterprise)
  profile_type = "app-store"
  profile_type = "ad-hoc" if values[:adhoc]
  profile_type = "development" if values[:development]
  profile_type = "enterprise" if enterprise

  Helper.log.info "Setting Provisioning Profile type to '#{profile_type}'"

  Actions.lane_context[SharedValues::SIGH_PROFILE_TYPE] = profile_type
end
```

I think `sigh` pattern makes profile_type priority ambiguous.
what do you think about this point?